### PR TITLE
Wayfinder init create location route

### PR DESCRIPTION
**Pull request by LocalNewsTV**
# Description

This PR includes the following proposed change(s):

- added new /locations end point with GET method
- Get method returns sample data based on Service BC Locations CSV (Subject to future change)
- created sample data to send (this PR is pre-mongo integrations)
- Added test for new endpoint
- updated swagger documentation to reflect new route

### Additional Note
- Plans exist to further optimize Swagger docs at a later date as schemas finalize and people are available in office for some knowledge transfer

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [x] Updating Testing Framework(s)
- [ ] Version change

## How Has This Been Tested?

- Ran in Docker
- Ran Jest tests
- Ran Manual Tests using Thunderclient
- Ran Tests in Swagger

## Frontend Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [x] New and existing unit tests pass locally with my changes
- [x] My changes includes Swagger documentation updates that reflect the changes I am introducing

### CODE REVIEW

File: src/api/models/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default as SingleLocation } from './singleLocation-model';
```
Comment (akroon3r) [inline src/api/models/index.ts:1]:
```suggestion
export { default as SingleLocation } from './singleLocation-model';

```

File: src/api/routes/index.ts [hunk 2]
```diff
@@ -1 +1,2 @@
-export { default as healthRouter } from './health-route';
+export { default as healthRouter } from './health-router';
+export { default as locationRouter } from './location-router';
```
Comment (akroon3r) [inline src/api/routes/index.ts:2]:
```suggestion
export { default as locationRouter } from './location-router';

```

File: src/api/routes/location-router.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import { getAllLocations } from '../controllers/location-controller';
+
+const router = express.Router();
+
+router.route('/locations')
+    .get(getAllLocations);
+
+export default router;
```
Comment (akroon3r) [inline src/api/routes/location-router.ts:9]:
```suggestion
export default router;

```

File: src/api/server.ts [hunk 4]
```diff
@@ -2,7 +2,7 @@ import app from './express';
 import dotenv from 'dotenv';
 dotenv.config();
 
-const port = process.env.PORT || 3006;
+const port = process.env.API_PORT || 3006;
```
Comment (akroon3r) [inline src/api/server.ts:5]: docker-compose has this listed as defaulted to 3000 I think? do we want to make these consistent?

**Review by akroon3r — APPROVED** [review]

File: src/api/models/index.ts (hunk 1 above)
Comment (Bottle7) [inline src/api/models/index.ts:1]: SingleLocation is already the default export in that file.
```suggestion
export SingleLocation from './singleLocation-model';
```

Comment (BradyMitch) [reply src/api/models/index.ts]: No { default as ... } is correct. If they were exporting a named export as SingleLocation then yes you would export your way, but in the SingleLocation file it is exported as default so it needs to be named here in the index.

File: src/api/models/index.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export SingleLocation from './singleLocation-model';
```
[the remark this answers is not in the document]
Comment (Bottle7) [reply src/api/models/index.ts:1]: Sorry

File: src/api/server.ts (hunk 4 above)
Comment (LocalNewsTV) [reply src/api/server.ts]: Feedback taken and addressed in latest commit

**Review by dallascrichmond — APPROVED** [review]

**Pull request merged**